tests/Unit/ProtoTests/BasicDiregapic/library_rest.proto

```diff
@@ -187,7 +187,7 @@ service Library {
     };
     option (google.api.method_signature) = "name,book";
     option (google.api.method_signature) =
-        "name,optional_foo,book,update_mask,physical_mask";
```

Comment:
removed `physical_mask` because this field does not exist in the protos, and so was throwing an exception. I believe this is the behavior we want.